di_website/templates/contactus/contact_page_landing.html

```diff
@@ -18,9 +18,14 @@
             </div>
             <div class="alert alert--success"><i role="presentation" aria-hidden="true" class="ico ico-success-success"></i>
                 <p><strong>{{page.success_alert}}</strong></p>
-                {% for attribute,value in form_submission.items %}
-                    <p>{{attribute|title}}: [ {{value}} ]</p>
-                {% endfor %}
+                {% with form_submission as content %}
+                    <p>Name: [ {{content.name | default:""}} ]</p>
+                    <p>Organisation: [ {{content.organisation | default:""}} ]</p>
+                    <p>Email: [ {{content.email|default:""}} ]</p>
+                    <p>Telephone: [ {{content.telephone | default:""}} ]</p>
+                    <p>Message:</p>
+                    <p>{{content.message | richtext}}</p>
+                {% endwith %}
             </div>
         </div>
     </div>
```

Comment:
Instead of hardcoded address include, could you use the `about` template tag to pull location snippets? https://github.com/devinit/DIwebsite-redesign/blob/develop/di_website/templates/about/who_we_are_page.html#L68-L82

Reply:
Yes, noted this in slack